components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue

```diff
@@ -1,24 +1,27 @@
 <template>
   <div class="gallery-activity-events-wrapper is-flex is-flex-direction-column">
-    <div class="events p-5 is-flex is-flex-direction-column">
+    <div class="events px-5 py-4 is-flex is-flex-direction-column">
       <div class="events-filter is-flex is-flex-wrap-wrap">
-        <div class="events-checkbox" @click="checkAll">
+        <div
+          class="is-clickable is-capitalized is-flex is-align-items-center"
+          @click="checkAll">
           {{ $t('tabs.tabActivity.all') }}
         </div>
-        <div
+        <label
           v-for="(value, name) in filters"
           :key="name"
-          class="events-checkbox">
+          class="is-clickable is-capitalized events-checkbox-container"
+          :class="cssActive(value)">
           <input
             :id="name"
             v-model="interactions"
             type="checkbox"
             :value="value"
             class="is-hidden" />
-          <label :for="name" :class="cssActive(value)">
+          <span :for="name" class="is-clickable">
```

Comment:
```suggestion
          <span :for="name">
```

it works without it:

![image](https://user-images.githubusercontent.com/22791238/224530868-1c4eeb8a-8bc5-41c4-8dd3-8b416acda083.png)